src/platforms/common/data-management/debug-files.mdx

```diff
@@ -214,6 +214,12 @@ traces.
 
 ### ProGuard Mappings
 
+<PlatformSection supported={["android"]}>
+<Note>
+  The recommended method for [Android users is to use the Gradle plugin](platforms/android/proguard/).
+</Note>
+</PlatformSection>
```

Comment:
```suggestion
<PlatformSection supported={["android"]}>

<Note>

  The recommended method for [Android users is to use the Gradle plugin](/platforms/android/proguard/).
  
</Note>

</PlatformSection>
```

Reply:
I guess we'd need spaces between them because the preview is broken, trying this out

Reply:
yep, you need spaces above and below the <Note></Note> syntax. Otherwise, it doesn't work as expected.